api/cql_server_test.cc

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (C) 2024-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+#ifndef SCYLLA_BUILD_MODE_RELEASE
+
+#include <seastar/core/coroutine.hh>
+#include <boost/range/algorithm/transform.hpp>
+
+#include "api/api-doc/cql_server_test.json.hh"
+#include "cql_server_test.hh"
+#include "transport/controller.hh"
+#include "transport/server.hh"
+#include "service/qos/qos_common.hh"
+
+namespace api {
+
+namespace cst = httpd::cql_server_test_json;
+using namespace json;
+using namespace seastar::httpd;
+
+struct connection_sl_params : public json::jsonable {
+    sstring _role_name;
+    sstring _workload_type;
+    sstring _timeout;
+
+    connection_sl_params(sstring role_name, sstring workload_type, sstring timeout)
+        : _role_name(role_name)
+        , _workload_type(workload_type)
+        , _timeout(timeout) {}
+    
+    std::string to_json() const override {
+        return fmt::format(R"({{"role_name":"{}", "workload_type":"{}", "timeout":"{}"}})", _role_name, _workload_type, _timeout);
```

Comment:
Hmm, I'm worried about escaping the strings here.

I see that `json::json_base` exists, which looks like a much safer interface for objects that need to be serialized. Can we use it here?